src/components/TextInput/TextInput.tsx

```diff
@@ -42,28 +44,37 @@ export const TextInput = React.forwardRef<HTMLSpanElement, TextInputProps>(funct
         name,
         value,
         defaultValue,
+        label,
         disabled = false,
         multiline = false,
         hasClear = false,
         error,
         autoComplete,
         onUpdate,
         onChange,
-        id,
+        id: originalId,
         tabIndex,
         style,
         className,
         qa,
+        controlProps: originalControlProps,
+        labelProps,
```

Comment:
Let's remove it for now. We'll be refactoring this later

Reply:
you mean remove both lines? in this case if someone using `controlProps` to set `style` attribute of `InputControl`, and using label, then the style will be overriden.

Reply:
I meant labelProps only

Reply:
ok, removed it for now. but for sure we shall need some alternative to adjust styles (as minimum, `max-width`) of label, as it will be needed in actual use places. in order not to forget, maybe open an issue for this?
cc @Marginy605